openmetadata-ui/src/main/resources/ui/playwright.config.ts

```diff
@@ -32,11 +32,18 @@ export default defineConfig({
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
+  workers: process.env.CI ? 2 : undefined,
```

Comment:
What should be the ideal value here?

Reply:
For CI, the number of workers can vary between 2 and 4. Currently, I have started testing with 2 workers. It's important to ensure that the number of workers does not exceed the number of CPU cores available on the CI machine. for now, 2 workers are running smoothly in CI.